### PR TITLE
DDF-2827 updated tests to use guest web context profile

### PR DESCRIPTION
**Pull request by stustison**
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@tbatie @ahoffer @brendan-hofmann @coyotesqrl

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by stustison** [comment]
Build will fail until the DDF change is merged.

File: distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java [hunk 1]
```diff
@@ -78,6 +78,8 @@ public void beforeAllianceTest() throws Exception {
             getServiceManager().startFeature(true, "nitf-render-plugin");
             getCatalogBundle().waitForCatalogProvider();
             configureSecurityStsClient();
+            configureRestForGuest();
+            getSecurityPolicy().waitForGuestAuthReady(REST_PATH.getUrl() + "?_wadl");
```
Comment (brendan-hofmann) [inline distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java]: If we're always waiting for the same path, perhaps it should be pushed down into the `waitForGuestAuthReady` method.

**Comment by bdeining** [comment]
Can we make a corresponding Alliance ticket and use a CAL-XX title for this?

File: distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java (hunk 1 above)
Comment (stustison) [inline distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java]: We may not be and that path may not always be available. This is just the general case.

**Pull request merged**